apps/core-app/src/renderer/src/modules/channel/channel-core.ts

```
@@ -2,20 +2,17 @@ import type {
+  StandardChannelData
 } from '@talex-touch/utils/channel'
-// eslint-disable-next-line @typescript-eslint/no-require-imports
+
 import type { IpcRendererEvent } from 'electron'
-import {
-  ChannelType,
-  DataCode,
-} from '@talex-touch/utils/channel'
+import { ChannelType, DataCode } from '@talex-touch/utils/channel'
 
 const { ipcRenderer } = require('electron')
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)